src/base/bittorrent/session.cpp

```diff
@@ -4119,18 +4124,51 @@ const CacheStatus &Session::cacheStatus() const
     return m_cacheStatus;
 }
 
-// Will resume torrents in backup directory
 void Session::startUpTorrents()
 {
+    qDebug("Initializing torrents resume data storage...");
+
+    const QString dbPath = Utils::Fs::expandPathAbs(
+                specialFolderLocation(SpecialFolder::Data) + QLatin1String("torrents.db"));
+    const bool dbStorageExists = QFile::exists(dbPath);
+
+    ResumeDataStorage *initialStorage = nullptr;
```

Comment:
would `previousStorage` be better?

Reply:
If it was just a matter of setting things up, I would agree. But it displays the fact that this is the "initial" storage, i.e. the storage from which we initially load data, but we will probably use another storage later (depending on the current settings).

Reply:
>the storage from which we initially load data

The suggestion was referring the storage as from "previous" session which seems match closely to the code actions here IMO. While "initial" seems more like setting up a whole new storage (and not related to existing storage) and could be ditched later.
This is a minor thing so you are free to pick you choice.

Reply:
We seem to have a subjective disagreement here.
Maybe just rename it to `storage`? Or `resumeDataStorage`?

Reply:
Or `startupStorage`?

Reply:
any of the suggested names will do.